website_chd_wishlist/static/src/js/website_chd_wishlist.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function () {
+
+
+	$('.favorites_option').submit( function(){
+		chosenform=this;
+			var elements = document.querySelectorAll( 'feedback *' );
+			var accessories
+			$('<input />').attr('type', 'hidden')
+         .attr('name', "summary")
+         .attr('value', document.getElementById("feedback").innerHTML)
+         .appendTo(chosenform);
+         return true;
+    });
+
+
+
+    $('.addwish').each(function () {
+        console.log(this);
+        var addwish = this;
+        $(addwish).on('click' , function ()
+        {
+            openerp.jsonRpc('/chd_init/wishadd/', 'call',
```

Comment:
I don't see where you react on this function. Also here, consider making this a model function